src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -131,6 +131,17 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     }
 
     /**
+     * Post differential inline comment
+     * @param json JSONObject to be sent to Harbormaster
+     * @return the Conduit API response
+     * @throws IOException if there is a network error talking to Conduit
+     * @throws ConduitAPIException if any error is experienced talking to Conduit
+     */
+    public JSONObject postInlineComment(JSONObject json) throws ConduitAPIException, IOException {
+        return conduit.perform("differential.createinline", json);
```

Comment:
This is, I think, the wrong conduit call. The PR / task at hand is to report **lint warnings**, not inline comments.